### PR TITLE
Update reusing-config.md

### PR DESCRIPTION
**Pull request by thomasr8**
Fixed some formatting errors in code samples.

# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

File: jekyll/_cci2/reusing-config.md [hunk 1]
```diff
@@ -459,10 +459,8 @@ commands:
 
 The following `enum` type declaration is invalid because the default is not declared in the enum list.
 
-```
-
 {% raw %}
-
+```
 yaml
```
Comment (michelle-luna) [inline jekyll/_cci2/reusing-config.md]: yaml has to be on the same line as the ticks

```yaml
version 2.1
```

File: jekyll/_cci2/reusing-config.md [hunk 2]
```diff
@@ -471,19 +469,16 @@ commands:
...
 
 #### Executor
 {:.no_toc}
 
 Use an `executor` parameter type to allow the invoker of a job to decide what executor it will run on.
 
-```
 {% raw %}
-
+```
```
Comment (michelle-luna) [inline jekyll/_cci2/reusing-config.md]: move yaml right after the tick marks

File: jekyll/_cci2/reusing-config.md [hunk 3]
```diff
@@ -859,22 +850,16 @@ workflows:
 
 ## Defining Conditional Steps
 
-Conditional steps allow the definition of steps that only run if a condition is
-met. 
+Conditional steps allow the definition of steps that only run if a condition isvmet.
```
Comment (michelle-luna) [inline jekyll/_cci2/reusing-config.md]: isvmet -> is met

File: jekyll/_cci2/reusing-config.md (hunk 1 above)
Comment (thomasr8) [reply jekyll/_cci2/reusing-config.md]: Fixed

File: jekyll/_cci2/reusing-config.md (hunk 3 above)
Comment (thomasr8) [reply jekyll/_cci2/reusing-config.md]: Fixed

File: jekyll/_cci2/reusing-config.md (hunk 2 above)
Comment (thomasr8) [reply jekyll/_cci2/reusing-config.md]: Fixed

**Comment by michelle-luna** [comment]
I think this was done in a separate PR?

**Pull request closed without merging**